### PR TITLE
Django Tutorial Part 8: User authentication and permissions

### PR DESCRIPTION
**Pull request by huylq-0414**

### CODE REVIEW

**Comment by huylq-0414** [comment]
ready

File: catalog/views.py [hunk 1]
```diff
@@ -54,3 +55,16 @@ def index(request):
...
+    template_name = "catalog/bookinstance_list_borrowed_user.html"
+    permission_required = "catalog.can_mark_returned"
+    paginate_by = 10
+
+    def get_queryset(self):
+        return (
+            BookInstance.objects.filter(borrower=self.request.user)
+            .filter(status__exact="o")
+            .order_by("-due_back")
+        )
```
Comment (huongnt-2545) [inline catalog/views.py]:
```suggestion
        return (
            BookInstance.objects.filter(borrower=self.request.user, status__exact="o")
            .order_by("-due_back")
        )
```

**Comment by huylq-0414** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
mớt pull nhé

**Comment by huongnt-2545** [comment]
merged

**Pull request merged**